hack/olm.sh

```diff
@@ -26,10 +27,12 @@ fi
 shift # shift position of the positional parameters for getopts
 
 # Options
-while getopts ":ic:" opt; do
+PRIVATE_KEY=""
+while getopts ":ic:k:" opt; do
     case "$opt" in
 	i) noCache="--image-build-args=\"--no-cache\"";;
 	c) OPERATOR_IMAGE="$OPTARG";;
+  k) PRIVATE_KEY="$OPTARG";;
```

Comment:
indentation

Reply:
What indentation do I need to fix? :-) Sorry not able to figure that out.

Reply:
I think it should be in line with i) and c)

Reply:
OK, some is not following tabs vs spaces consistently. This file is a mix of mainly spaces and some tabs. I use a tab here and fix tab and spaces in a separate PR.

Reply:
Fixed. PTAL.